ansible_collections/arista/avd/examples/l2ls-fabric/documentation/devices/LEAF1.md

```diff
@@ -50,19 +50,19 @@
 
 | Management Interface | Description | Type | VRF | IP Address | Gateway |
 | -------------------- | ----------- | ---- | --- | ---------- | ------- |
-| Management0 | OOB_MANAGEMENT | oob | MGMT | 172.16.100.105/24 | 172.16.100.1 |
+| Management1 | OOB_MANAGEMENT | oob | MGMT | 172.16.100.105/24 | 172.16.100.1 |
```

Comment:
why management 1? I think it makes sense to keep cEOS-lab as the default platform. The reason is - examples will be most commonly tested on cEOS-lab. Please keep everything in a way that can be "instantly" deployed on cEOS-lab.